### ISSUE TITLE
Can't build oryx 8.0

### ISSUE THREAD

**Issue by stiv-yakovenko** [OryxProject/oryx#355]
pom.xml references oryx-common 8.0, which is not on repository: https://mvnrepository.com/artifact/com.cloudera.oryx/oryx-common

![image](https://user-images.githubusercontent.com/4163254/45915945-17449380-be88-11e8-9d67-0d15499e3d62.png)

It was building perfectly when version was 7.1....

**Comment by stiv-yakovenko** [comment]
If I download 2.7.1 release building with maven 3.5.4 gives me this:

**mvn -Drat.skip=true -Dmaven.test.skip=true package -Dcheckstyle.skip**

![image](https://user-images.githubusercontent.com/4163254/45916115-d13cff00-be8a-11e8-80bf-9b0b76c2416a.png)

Or if I call the same command from ./app/oryx-app-serving, then I get these complaints:

![image](https://user-images.githubusercontent.com/4163254/45916132-27aa3d80-be8b-11e8-9116-b431468bdba7.png)

**Comment by stiv-yakovenko** [comment]
Tried with expected maven version, same problem:

![image](https://user-images.githubusercontent.com/4163254/45916243-1d893e80-be8d-11e8-8e96-d061eef602c1.png)

**Comment by srowen** [comment]
It builds correctly as is. Build from the top level. Otherwise you are building only part of the project. This is not how Maven works.

**Comment by stiv-yakovenko** [comment]
If I build from root folder, I get these scala-related complaints:

![image](https://user-images.githubusercontent.com/4163254/45920014-c5702d80-bec8-11e8-8c95-a24b918812fa.png)

**Comment by srowen** [comment]
As I say, the build clearly works as-is. See the Travis output above. It sounds like you have modified the project in a few ways, and I suspect that is the problem. This is not a project question.

**Comment by stiv-yakovenko** [comment]
You are right, I've redownloaded source and it has built sucessfully on my windows machine. Excuse me please for missleading.
